tests/test_shell_tools.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: MIT
+
+import unittest
+from argparse import Namespace
+
+import odxtools.cli.browse as browse
+import odxtools.cli.find as find
+import odxtools.cli.list as list
```

Comment:
`list` is a bad name because it conflicts with the built-in `list()` function to generate lists. `odxtools_list`?